developer-wsl-info.md

```diff
@@ -0,0 +1,68 @@
+# Develop in Millicast SDK with Windows Subsystem for Linux (WSL)
+
+## Table of Contents
+- Millicast SDK for WSL
+  - [Running test](#running-test)
+  
+## Running Test
+After completing the general Millicast SDK setup instructions listed in the [developer-info.md](developer-info.md) file, WSL users may experience multiple issues when trying to run test locally.
+
+### First Issue
+ The first issue you may encounter is a SyntaxError occuring at the Puppeteer/util/disposable.js file which will appear in your terminal as an error due to an unexpected "??=" token.
+
+ The quick way to resolve this error is by using "nvm use v16.14.0" or any higher version of Node.
+
+ -----------------
+### Second Issue
```

Comment:
nitpick : A better name is something along the lines of what you see in the console :)